src/Microsoft.Health.Fhir.Api/Controllers/FhirController.cs

```diff
@@ -145,7 +145,7 @@ public IActionResult CustomError(int? statusCode = null)
         /// <param name="resource">The resource.</param>
         [HttpPost]
         [Route(KnownRoutes.ResourceType)]
-        [AuditEventSubType(AuditEventSubType.Create)]
+        [AuditEventType(AuditEventSubType.Create)]
```

Comment:
Should the enum be changed to `AuditEventType` also?

Reply:
The ValueSet itself is defined by http://hl7.org/fhir/STU3/valueset-audit-event-sub-type.html, which is called AuditEventSubType so I left it as is.